api/src/main/kotlin/com/nikitahohulia/listeningplatform/controller/nats/user/GetUserByIdNatsController.kt

```diff
@@ -0,0 +1,42 @@
+package com.nikitahohulia.listeningplatform.controller.nats.user
+
+import com.google.protobuf.Parser
+import com.nikitahohulia.listeningplatform.controller.nats.NatsController
+import com.nikitahohulia.listeningplatform.dto.response.toProto
+import com.nikitahohulia.listeningplatform.service.UserService
+import com.nikitahohulia.nats.NatsSubject
+import com.nikitahohulia.nats.commonmodels.user.User
+import com.nikitahohulia.nats.reqreply.user.get_by_id.proto.GetUserByIdRequestCommon
+import com.nikitahohulia.nats.reqreply.user.get_by_id.proto.GetUserByIdResponseCommon
+import io.nats.client.Connection
+import org.springframework.stereotype.Component
+
+@Component
+class GetUserByIdNatsController(
+    override val connection: Connection,
+    private val userService: UserService
+) : NatsController<GetUserByIdRequestCommon, GetUserByIdResponseCommon> {
+    override val subject = NatsSubject.User.GET_BY_ID
+    override val parser: Parser<GetUserByIdRequestCommon> = GetUserByIdRequestCommon.parser()
+
+    override fun handle(request: GetUserByIdRequestCommon): GetUserByIdResponseCommon = runCatching {
+        val user = userService.getUserById(request.userId)
+
+        buildSuccessResponse(user.toProto())
```

Comment:
```suggestion
        buildSuccessResponse(userService.getUserById(request.userId).toProto())
```

Reply:
Not really understand the reason to do so. After all that is the test for nats controller, not UserService